src/Middleware/StaticFiles/test/FunctionalTests/StaticFileMiddlewareTests.cs

```diff
@@ -191,6 +191,7 @@ public async Task HeadFile_HeadersButNotBodyServed(string baseUrl, string baseDi
                     webHostBuilder
                     .ConfigureServices(services => services.AddSingleton(LoggerFactory))
                     .UseKestrel()
+                    .UseUrls(TestUrlHelper.GetTestUrl(ServerType.Kestrel))
```

Comment:
Why not KestrelServerOptions.ListenLocalhost(0)?

Reply:
That's about what ends up happening under the hood:
https://github.com/dotnet/aspnetcore/blob/52eff90fbcfca39b7eb58baad597df6a99a542b0/src/Hosting/Server.IntegrationTesting/src/Common/TestUriHelper.cs#L35

Reply:
But a few tests actually use a different server:
https://github.com/dotnet/aspnetcore/blob/91a13d4f24f3c56cd8666cf45219ad8327a4c98b/src/Middleware/StaticFiles/test/FunctionalTests/StaticFileMiddlewareTests.cs#L275